content/mqtt/index.textile

```diff
@@ -0,0 +1,113 @@
+
+bc[javascript]. {
+  var ably = new Ably.Realtime('REPLACE_WITH_YOUR_API_KEY');
+  var decoder = new TextDecoder();
+  var channel = ably.channels.get('input');
+  channel.subscribe(function(message) {
+    var command = decoder.decode(message.data);
+  });
+}
+
+In the above example, @command@ will now contain the message in its original string form.
+
+h2(#use-cases). Recommended usage
+
+We recommend you use MQTT to interact with Ably when:
+
+* Using devices which do not have a native Ably client library, such as Arduino platforms, C/C++ applications, and so on
+* If bandwidth is limited and you want to keep network traffic to a minimum
+
+h2(#supported). Supported features
+
+Our MQTT adapter only supports features supported by both the MQTT protocol and the Ably platform. So:
+
+* It only supports MQTT 3.1.1 clients only. Connection attempts using earlier protocol versions will be rejected
+* Publishing supports QoS 0 or 1. Subscribing only supports QoS 0
+* Session resumption is supported within the usual "Ably time limit of two minutes":https://support.ably.io/solution/articles/3000044639-connection-state-recovery
+* It doesn’t support any MQTT features that aren’t normally supported by Ably, such as @WILL@ messages, the @RETAIN@ flag or "wildcard channel subscriptions":https://support.ably.io/solution/articles/3000063123-can-i-attach-and-subscribe-to-channels-using-wildcards
+* It doesn't support any Ably features that aren't supported by the MQTT protocol, such as presence, history and push notifications (though presence may be supported soon). Of course, you can always use the "Ably REST API":https://www.ably.io/documentation/rest-api in conjunction with the MQTT adapter to access features available over REST
+
+h2(#security). Security
+
+We recommend that you always make use of "token authentication over basic authentication":https://support.ably.io/support/solutions/articles/3000038607-token-and-basic-authentication when trying to connect from devices you may not trust. In addition, if you're using basic authentication you'll be required to use SSL to connect to our adapter to ensure the API key cannot be accessed by someone listening in.
+
+h3(#token-auth). Token authentication
+
+For token authentication you'll be required to "create a token from your own servers":https://www.ably.io/documentation/general/authentication#token-authentication. Once you have the token, you can simply pass it through when trying to connect to Ably as the connection's @username@, leaving the @password@ empty. In the future we will be adding "JWT":https://jwt.io/ (JSON Web Tokens) as a form of authentication.
```

Comment:
Replace "we will be adding JWT as a form of authentication" with "we will be extending our Token authentication scheme to support JWT" ?